src/layers/core/path-layer/path-layer-vertex.glsl.js

```diff
@@ -150,6 +154,14 @@ vec3 lineJoin(vec3 prevPoint, vec3 currPoint, vec3 nextPoint) {
 
   vCornerOffset = offsetVec * offsetDirection * offsetScale;
 
+  // Generate variables for dash calculation
```

Comment:
FP64 layers still involve tedious copying I see. Makes me wonder if we should have a `-vertex-common.glsl.js`. There must be some way to consolidate this, I'll try to think of something.